src/database/modele/Question.js

```diff
@@ -24,14 +26,27 @@ const QuestionSchema = new mongoose.Schema({
     refDiscussion:{
         type: Number,
         ref: 'Discussion',
-        required: true
+        required: true,
+        //Validate will permit us to make some validation before a refDiscussion is saved
+        //The function check if the id of the value passed exits in the DataBase if yes it will return true otherwise false
```

Comment:
```suggestion
        // Validate will permit us to make some validation before a refDiscussion is saved
        // The function check if the id of the value passed exits in the DataBase if yes it will return true otherwise false
```